src/App.jsx

```diff
@@ -23,6 +44,7 @@ function App() {
 {/* THis is how you add a comment. Also imported TodoList */}
 
   return (
+    /* Not sure if I was supposed to use "<React.Fragment>" of f the short hand version "<> and </>" is fine so I left it */
```

Comment:
either way it's okay, at my work the linter is configured to make us use <Fragment>. So it really depends how you want your code to look like.